03/implement-hamming.checkpoint.md

```diff
@@ -90,3 +90,59 @@ def hamming_distance(strand1, strand2):
 
 ### !end-challenge
 <!-- prettier-ignore-end -->
+
+### !challenge
+
+* type: short-answer
+* id: de535a7c-8d40-43e0-8214-038fdab47fd0
+* title: What is the time complexity of your solution?
+* points: 1
+* topics: Big-O, Python, lists, matrices
+
+##### !question
+
+What is the time complexity of your solution?
```

Comment:
```suggestion
What is the time complexity of your solution? Explain.
```

Reply:
I think it would be good for students to write a short explanation. But I'll leave that up for you to decide. If you like the suggestion, could you add it to the other PSEs?